Description: Fix the I2Util paths
Origin:      upstream
Forwarded:
Author:      Dominic Hamon <dominic@google.com>
Last-Update: 2013-10-02
--- a/configure	2013-09-24 06:49:41.000000000 -0700
+++ b/configure	2013-10-02 11:51:50.211429000 -0700
@@ -2434,14 +2434,15 @@ I2UTILINCS=""
 if test "$with_I2util" != "no"; then
 	# first, check for directory given.
 	if test "$with_I2util" != "yes"; then
-		I2util_dir=`dirname $with_I2util`
+		I2util_dir=$with_I2util
 		case $I2util_dir in
 			/*) ;; # already an absolute path
 			*) I2util_dir="`pwd`/$I2util_dir" ;;
 		esac
-		I2UTILINCS="-I$I2util_dir/include $I2UTILINCS"
-		I2UTILLDFLAGS="-L$I2util_dir/lib $I2UTILLDFLAGS"
-		I2UTILLIBDEPS="$I2util_dir/lib/libI2util.a"
+		I2util_include_dir=`dirname $with_I2util`/include
+		I2UTILINCS="-I$I2util_include_dir/include $I2UTILINCS"
+		I2UTILLDFLAGS="-L$I2util_dir $I2UTILLDFLAGS"
+		I2UTILLIBDEPS="$I2util_dir/libI2util.a"
 	# now, check for sub-build/sub-configure
 	elif test -d I2util/I2util; then
 
